build.gradle

```diff
@@ -1,12 +1,18 @@
 buildscript {
+    ext {
+        support_version="24.2.0"
+    }
+
     repositories {
         jcenter()
         maven { url "https://jitpack.io" }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.5.0'
```

Comment:
I would commit the update of the individual dependencies in separate commits. The updates are kind of hidden here.